editor/src/core/workers/parser-printer/parser-printer-parsing.ts

```diff
@@ -685,7 +671,7 @@ function parseOtherJavaScript<E extends TS.Node, T>(
         currentScopeInner: Array<string>,
         statement: TS.Statement,
       ): Array<string> {
-        if (TS.isFunctionDeclaration(statement) && statement.name != null) {
+        if (TS.isFunctionLike(statement) && statement.name != null) {
```

Comment:
Handy!

Reply:
This caught me about because apparently there is also `TS.isMethodDeclaration`, which of course is completely different to a function declaration since it applies to a class